basic_hashtable/b_hashtables.c

```diff
@@ -94,7 +106,9 @@ void hash_table_insert(BasicHashTable *ht, char *key, char *value)
  ****/
 void hash_table_remove(BasicHashTable *ht, char *key)
 {
-
+  int hashed = hash(key, ht->capacity);
```

Comment:
What happens if the user tries to remove an entry that doesn't exist?
also, we have `destroy_pair` - that's the right tool to use instead of `free`